src/App.tsx

```diff
@@ -141,6 +140,22 @@ const getL1TxnReceipt = async (txnHash: string) => {
   }
 };
 
+const isArbInboxEthDeposit = async(
```

Comment:
We should probably add a note saying this doesn't detect all eth deposits

Reply:
other people looking into this repo for inspiration might shoot themselves on the foot

Reply:
updated detection method